docs/tutorials/web.md

```diff
@@ -5,10 +5,400 @@ has_children: false
 nav_order: 7
 ---
 
-# Deploy ML models on the web
 
- Check out the [release blog here](https://cloudblogs.microsoft.com/opensource/2021/09/02/onnx-runtime-web-running-your-machine-learning-model-in-browser/)
+# Deploy and inference ML models on the web
```

Comment:
```suggestion
# Classify images in a web application
```

Reply:
I am not sure I agree with the change. What is the reason that you think the title should be changed? I was thinking about SEO and the title of the nav (which was there prior to this doc)

Reply:
I'm thinking we'll end up with a few specific web tutorials. With the more generic title, the implication that this is a more generic tutorial